build.gradle

```diff
@@ -1,10 +1,20 @@
 apply plugin: 'groovy'
+apply plugin: 'java'
 
 dependencies {
     compile gradleApi()
     compile localGroovy()
 }
 
-sourceCompatibility = '1.7'
-targetCompatibility = '1.7'
+sourceCompatibility = '1.10'
+targetCompatibility = '1.10'
 
+tasks.withType(AbstractArchiveTask) {
+    preserveFileTimestamps = false
+    reproducibleFileOrder = true
+}
+
+jar.doLast { task ->
+    ant.checksum file: task.archivePath
+    println "md5=" + file("build/libs/gradle-witness.jar.MD5").text.trim()
```

Comment:
I'm producing a different checksum result locally than what you report at https://github.com/bisq-network/bisq/pull/1901 in the commit comment on https://github.com/bisq-network/bisq/pull/1901/commits/ee968220640c8586860eb12835d6e945ac3bfa2e:

```
~/Work/bisq-network/gradle-witness[update-gradle-build]
$ git log -1 --oneline
a4a1b2d (HEAD -> update-gradle-build, devinbileck/update-gradle-build) Update source/target compatibility to 1.10

$ ./gradlew --version
------------------------------------------------------------
Gradle 4.10.2
------------------------------------------------------------

Build time:   2018-09-19 18:10:15 UTC
Revision:     b4d8d5d170bb4ba516e88d7fe5647e2323d791dd

Kotlin DSL:   1.0-rc-6
Kotlin:       1.2.61
Groovy:       2.4.15
Ant:          Apache Ant(TM) version 1.9.11 compiled on March 23 2018
JVM:          10.0.2 ("Oracle Corporation" 10.0.2+13)
OS:           Mac OS X 10.12.5 x86_64

$ ./gradlew clean build
> Task :compileGroovy
> Task :jar
md5=ae4796f320ef3200515183fa9d3f4759

BUILD SUCCESSFUL in 2s
4 actionable tasks: 4 executed

$ cat ./build/libs/gradle-witness.jar.MD5
ae4796f320ef3200515183fa9d3f4759
```

(your commit comment says you produce the md5 sum `237be11ebb7604afa5ff478ec33d1df5`)

Reply:
Argg! I will investigate further.

Reply:
It is due to different line endings in the witness.properties file. If I change it to LF (vs CRLF) I get the same result as you. Shall I add a `.gitattributes` file to ensure consistent line endings and prevent issues like this in the future?